src/components/presentations/UserProfile/__tests__/Userprofile.test.jsx

```diff
@@ -54,10 +56,12 @@ describe('ProfilePage component', () => {
   it('should change state', () => {
     const wrapper = shallow(
       <Userprofile
-        getProfile={() => 'profile'}
+        getProfile={jest.fn()}
         userProfile={{ userProfile: { profile: { fake: 'data' } } }}
         articles={{ articleData: [{ author: { first_name: 'sam' } }] }}
```

Comment:
We should be defining mock data like this inside mock files and import them wherever we need to use them. Please create a `mocks` folder and put this there @sojida